### PR TITLE
[Task-10][Backend] As a user, I can change password in the settings

### LINKED ISSUE #10: [Task-10][Backend] As a user, I can change password in the settings
## Why

The user should be able to change the password in the settings
 
## Acceptance Criteria
- [ ] User can change password

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/elixir_search_extractor/projects/1#card-61390405

## What happened 👀

Add user settings page with password change functionality.

## Insight 📝

- Add user settings page
- Password change functionality

**Note: Removed all traces of confirmation, reset password and email change as sending email requires some mailer credentials and it is not in the scope of this project.** 
 
## Proof Of Work 📹

ElixirSearchExtractor · Phoenix Framework (3)

### CODE REVIEW

File: lib/elixir_search_extractor/accounts.ex [hunk 1]
```diff
@@ -6,7 +6,7 @@ defmodule ElixirSearchExtractor.Accounts do
   import Ecto.Query, warn: false
   alias ElixirSearchExtractor.Repo
   # credo:disable-for-next-line Credo.Check.Consistency.MultiAliasImportRequireUse
```
Comment (rosle) [inline lib/elixir_search_extractor/accounts.ex]: I'm thinking about disable this check in the credo config cause you've mentioned that it has issue, right?. I saw in credo repo itself also check this rule off as default - https://github.com/rrrene/credo/blob/master/.credo.exs#L167, what do you think? 💡

**Review by rosle — APPROVED** [review]
👍

File: lib/elixir_search_extractor/accounts.ex (hunk 1 above)
Comment (rafayet-monon) [reply lib/elixir_search_extractor/accounts.ex]: I agree, added in [dc300b9](https://github.com/rafayet-monon/elixir_search_extractor/pull/27/commits/dc300b93812d683808a3d774ff10dc948faf64eb)

File: test/elixir_search_extractor_web/features/settings_page/settings_page_test.exs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+defmodule ElixirSearchExtractorWeb.SettingsPage.SettingsPageTest do
+  use ElixirSearchExtractorWeb.FeatureCase
```
Comment (hanam1ni) [inline test/elixir_search_extractor_web/features/settings_page/settings_page_test.exs]:
```suggestion
  use ElixirSearchExtractorWeb.FeatureCase, async: true
```

Comment (rafayet-monon) [reply test/elixir_search_extractor_web/features/settings_page/settings_page_test.exs]: Added in [6d138e0](https://github.com/rafayet-monon/elixir_search_extractor/pull/27/commits/6d138e0797e370ce45a98cf9233ad84d6400653b)

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**